cardinal/server/server.go

```diff
@@ -144,6 +146,11 @@ func setupRoutes(app *fiber.App, engine *ecs.Engine, cfg Config) {
 		msgs[msg.Group()][msg.Name()] = msg
 	}
 
+	// Route: /events/
+	websocketUpgrader, websocketHandler := handler.WebSocketEvents(eventHub)
+	app.Use("/events", websocketUpgrader)
```

Comment:
For the sake of my understanding, is basically saying that we use the upgrader for everything other than GET requests to `/events`?

Reply:
The api is not clear here. This is what they did in the docs. 

Basically there's two parts of a websocket connection. The http connection and then an upgrade to websockets. The way fiber does it is via context.Next(). Within the upgrader it will pass it along to the websocket handler. Why it's Use and Get is because the api designers decided to do it this way. It's an arbitrary decision from them. Likely done to make it fit the shape of existing methods and type checks.